libs/dh/market-participant/grid-areas/overview/src/lib/dh-market-participant-gridarea-overview.component.ts

```diff
@@ -109,18 +131,29 @@ export class DhMarketParticipantGridAreaOverviewComponent {
 
     const columnsPath = 'marketParticipant.gridAreas.columns';
 
+    const statusPath = 'marketParticipant.gridAreas.status';
+    const typesPath = 'marketParticipant.gridAreas.types';
+
     const headers = [
       `"${translate(columnsPath + '.code')}"`,
       `"${translate(columnsPath + '.actor')}"`,
       `"${translate(columnsPath + '.organization')}"`,
+      `"${translate(columnsPath + '.type')}"`,
+      `"${translate(columnsPath + '.status')}"`,
     ];
 
     const lines = dataSorted.map((gridArea) => [
       `"${gridArea.code}"`,
       `"${gridArea.actor}"`,
       `"${gridArea.organization}"`,
+      `"${translate(typesPath + '.' + gridArea.type)}"`,
+      `"${translate(statusPath + '.' + gridArea.status)}"`,
     ]);
 
     exportToCSV({ headers, lines, fileName: 'grid-areas' });
   }
+
+  public onGridAreaTypeChange(type: GridAreaType | null): void {
+    this.selectedGridAreaType.set(type);
+  }
```

Comment:
suggestion: looks like this can be skipped because the `selectedGridAreaType` is read in the `effect` thus making it a dependency. So the `effect` will re-run when the Signal is updated by `ngModel` in the template.